SlevomatCodingStandard/Sniffs/Operators/NegationOperatorSpacingSniff.php

```diff
@@ -0,0 +1,99 @@
+<?php declare(strict_types = 1);
+
+namespace SlevomatCodingStandard\Sniffs\Operators;
+
+use PHP_CodeSniffer\Files\File;
+use PHP_CodeSniffer\Sniffs\Sniff;
+use SlevomatCodingStandard\Helpers\TokenHelper;
+use function in_array;
+use function strlen;
+use const T_CLOSE_PARENTHESIS;
+use const T_CLOSE_SHORT_ARRAY;
+use const T_CLOSE_SQUARE_BRACKET;
+use const T_CONSTANT_ENCAPSED_STRING;
+use const T_DNUMBER;
+use const T_ENCAPSED_AND_WHITESPACE;
+use const T_LNUMBER;
+use const T_MINUS;
+use const T_NUM_STRING;
+use const T_STRING;
+use const T_VARIABLE;
+use const T_WHITESPACE;
+
+final class NegationOperatorSpacingSniff implements Sniff
+{
+
+	public const CODE_INVALID_SPACE_AFTER_MINUS = 'InvalidSpaceAfterMinus';
+
+	/** @var bool */
+	public $requireSpace = false;
```

Comment:
Please change to integer to be consistent with other sniffs.

Reply:
You mean to change it to "requiredSpacesCount"? You sure that makes sense? Or what other sniff uses integer instead of boolean, and why? Doesn't make sense to me.

Reply:
Yes, I mean `$spacesCount`. I use always integer so I want to be consistent with other sniffs.